scriptmodules/libretrocores/lr-flycast.sh

```diff
@@ -30,8 +30,14 @@ function sources_lr-flycast() {
 function build_lr-flycast() {
     make clean
     if isPlatform "rpi"; then
+        local params=()
```

Comment:
Just a small extra change. The `local` declarations should be in the main function body scope, not inside `if` blocks. Otherwise `params` will not be initialised if `isPlatform "rpi"` is `false`.
Simply move the local declaration to the beginning of the function and that will do it.

Reply:
but params are not used if not rpi?

Reply:
Oh you are right, didn't notice that the `make` invocation is inside the `if` block and there is yet *another* `make` invocation later down below. That maybe can be improved but I agree it is not the purpose of this PR, so I won't bug about it.

In bash there is not really strong scoping, i.e. no `if` block scoping, so putting `local`s at the beginning of the functions is more good practice for legibility, i.e. "hey this function has the following locals", than required programming syntax.

The majority (if not all?) of scriptmodules in RetroPie using `local params=()` declare it at the beginning of functions so I would stick to that for now.